### DISCUSSION TITLE
[Issue] NPM Package 'reown' Not Found  In Registiry - Error E404

### DISCUSSION THREAD

**Discussion by bezata** [WalletConnect/walletconnect-monorepo#5412] [Developer Support] [answered]
I encountered an issue while trying to install the 'reown' package using npm and bun. The installation failed with a 404 error indicating that the package is not found in the npm registry.  Below are the error details:
`npm ERR! code E404.`
`npm ERR! 404 Not Found - GET https://registry.npmjs.org/reown - Not found`
`npm ERR! 404 `
`npm ERR! 404  'reown' is not in this registry.`
`npm ERR! 404 `
`npm ERR! 404 Note that you can also install from a`
`npm ERR! 404 tarball, folder, http url, or git url.`

**Comment by rtomas** [comment]
How are you attempting to install it?

**Reply by bezata** [reply]
> How are you attempting to install it?
Just as at the docs i tried bun add @reown/appkit @reown/appkit-adapter-wagmi wagmi viem @tanstack/react-query
or npm version of it also and when i trying to get request from https://registry.npmjs.org/reown it says not found. Is it only happening to me or is it general?

**Comment by bezata** [comment, marked as answer]
In large projects, things may slip your mind when everything is overwhelming. If anyone else is facing this issue, simply deleting `reown : ""` from the package.json, where there's no version specified, should be enough.
